wgsl/index.bs

```diff
@@ -6941,21 +6941,21 @@ To declare a variable for accessing a particular output built-in *Y* from an ent
   * Otherwise, make the result type to be a structure, where one of the fields is the new variable.
 * Apply a `builtin(`*Y*`)` attribute to the result variable.
```

Comment:
This section seems wrong as builtin can't be applied to a variable

Reply:
Yeah.  I should have read this section more closely.  It needs more rework.